Algorithms/Stack/InfixToPostfix.java

```diff
@@ -0,0 +1,70 @@
+
+import java.util.*;
```

Comment:
Doesn't import.java.*; solve all the purpose of all those imports?

Reply:
All the requested changes is applied please review.

Reply:
> Doesn't import.java.*; solve all the purpose of all those imports?

No, as we have to mention the package name as well.